content/blog/2020-05-08-dvc-ambassador-program.md.md

```diff
@@ -0,0 +1,151 @@
+the same time, it means going farther to reach people outside the organization-
+including people who don’t know about DVC yet, people who need some help getting
+started, and people who are already excited about our mission and want to find
+meaningful ways to pitch in.
+
+It’s challenging, no doubt, but being an ambassador can be an amazing
+experience! Before becoming DVC’s first official ambassador, I served as a
+[Fedora ambassador](https://fedoraproject.org/wiki/User:Mribeirodantas) for
+several years. I got a lot out of it: I participated in lots of events, made
+friends, and traveled. Every single time I’ve contributed to an open source
+project, I got this nice feeling that it was all worth it. If you agree with the
+core values of the project, a great relationship lies ahead :).
+
+## About Iterative and DVC
+
+DVC got started in 2017 as a personal project by Dmitry Petrov (on May 4,
+[we celebrated our 3rd birthday](https://dvc.org/blog/dvc-3-years-and-1-0-release)).
+In 2018, Dmitry teamed up with his co-founder Ivan Shchenklein to incorporate
+Iterative.ai and grow the project. Currently, Iterative.ai’s flagship project is
+Data Version Control (DVC), an open source version control system for managing
+complex workflows, datasets, and models. Development is ongoing in the core DVC
+project as well as new ventures into
+[MLOps and Continuous Integration & Delivery (CI/CD)](https://dvc.org/blog/reimagining-devops-video)
+for data science. The team is small-and-mighty, with developers, engineers, and
+data scientists on four continents. The open source community is a huge part of
+all Iterative.ai projects; currently, DVC has more than
+[5,000 stars on GitHub](https://github.com/iterative/dvc) and more than 100
+individual contributors!
+
+One of DVC’s main principles is adapting existing software engineering practices
+to machine learning. For example, DVC is built around Git version control: in an
+ML project using DVC, each experiment corresponds to a Git commit. When you
+check out any commit, you’ll see the source code as it was when you made the
+commit- as expected. But, you’ll also see your datasets as they were and the
+exact pipeline of commands you ran in that experiment!
+
+## How to become an ambassador
+
+If you’re interested in becoming an ambassador, send us an email at info@dvc.org
+with the subject line “I want to be an ambassador!” Please tell us:
+
+- A little about yourself and your professional background
+- Any outreach work you’ve done before
+- What kind of ambassador activities you’d be most interested in participating
+  in
+
+The program is structured to provide a lot of flexibility, so each ambassador
+can do outreach in ways that are personally motivating and enjoyable. There are
+a few guidelines:
+
+- We ask for a one-year commitment
+- We ask ambassadors to contribute four activities per year, about once every
+  three months.
```

Comment:
It does not sound like a good amount of work (for ambition folks). `four activities per year` --> `from four to eight activities per year`

Reply:
how about `at least four activities a year`?

Reply:
Maybe eight doesn't sound either hehe. What about "at least four activities per year" (no upper limit)?